app/components/forms/wizard/wizard-footer.hbs

```diff
@@ -22,6 +22,7 @@
       </button>
     {{/if}}
     {{#if (eq @event.state 'draft')}}
+      <div class="mt-8 {{if device.isMobile }}"></div>
```

Comment:
{{if device.isMobile }} why this is here when it's for nothing

Reply:
And build is failing too due to this

Reply:
> And build is failing too due to this

@maze-runnar so the button is overlapping in mobile view , that's why i add can you tell what i do

Reply:
{{if device.isMobile 'mt-8'}}